src/utils.js

```diff
@@ -64,6 +64,26 @@ function isParametersMatching(parameters, required) {
 }
 
 function isBodyMatching(body, requiredBody) {
+  if (requiredBody instanceof FormData) {
```

Comment:
`instanceof` checks won't work as only one specific version of the module would be handled.
This would probably need some kind of duck typing.
Also, `FormData` isn't global in node, so this will break the module if there's a dependency on it.